gql-queries/vol_qry.R

```diff
@@ -0,0 +1,39 @@
+# Problem 4b
+# vol_qry(id, from, to)
+
+vol_qry <- function(id, from, to){
+  glue::glue(.open = "{{", .close = "}}", '{
+  trafficData(trafficRegistrationPointId: "{{id}}") {
+    volume {
+      byHour(from: "{{from}}", to: "{{to}}") {
+        edges {
+          node {
+            from
+            to
+            total {
+              volumeNumbers {
+                volume
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}'
+)
+}
+
+# I do not understand why it does not work with the general code, 
+# but this way with specific numbers it does works
+GQL(
+  vol_qry(
+    id="97411V72313", 
+    from="2022-05-01T06:55:47Z",
+    to="2022-05-08T06:55:47Z"
+  ),
+  .url = configs$vegvesen_url
```

Comment:
The reason this fails is because you have used the cat() function in the to_iso8601() function. 

If you look at the help function for cat(), you see that the returned value is NULL. This means, when you are using it to return a value, even though the output seems to be correct, the actual returned value is NULL. To fix this, you could have just written "paste0(iso8601(date), "Z")", excluding the cat() function.